teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './person'
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, *args, **kwargs)
+    super(*args, **kwargs)
```

Comment:
- [ ] Here you should pass the attributes that the `Person` class is asking for, kindly change this 😄, it should be something like this:
```ruby
super(age, name, parent_permission)
```

Reply:
done :+1: